tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -839,6 +841,7 @@ def get_defaults(cls):
 
 @RunIf(min_cuda_gpus=1)  # trigger this test on our GPU pipeline, because we don't install the package on the CPU suite
 @pytest.mark.skipif(not package_available("lightning_colossalai"), reason="Requires Colossal AI Strategy")
+@pytest.mark.skip
```

Comment:
This is skipped here, since the validation for the precision argument is hardcoded in https://github.com/Lightning-AI/lightning-colossalai . I tried to first change it there, but these changes won't become effective until the strategy was released to pypi again.

After this was merged, I will post a follow-up PR in https://github.com/Lightning-AI/lightning-colossalai which will then be released and then have a follow-up PR to again add this test.

Discussed offline with @awaelchli